packages/hint-amp-validator/tests/tests.ts

```diff
@@ -38,4 +38,4 @@ const defaultTests: HintTest[] = [
     }
 ];
 
-hintRunner.testHint(hintPath, defaultTests);
+hintRunner.testHint(hintPath, defaultTests, { parsers: ['html'] });
```

Comment:
I think we should modify the hintRunner to add the HTML parser by default to avoid having to do this.

Reply:
I have been talking with Tony, and I'm going to move the traverse stuff to a utils, instead of being part of the parser.

The parser will just parse. So this will not be a dependency anymore.

Reply:
FTFY 😛
> The parser will just ~~parse~~ traverse.